### PR TITLE
DOC-795: Workaround for EditorManager API docs marked as private

### PR DESCRIPTION
**Pull request by tylerkelly13**
Related Ticket: DOC-795

Description of Changes:
* Workaround for EditorManager API docs marked as private
* Remove tinymce.data namespace from api docs build (APIs removed in 4.7)

Pre-checks:
* [x] Changelog entry added
* [x] Tests have been added (if applicable)
* [x] Branch prefixed with `feature/` for new features (if applicable)
* [x] License headers added on new files (if applicable)

Review:
* [x] Milestone set
* [x] Review comments resolved

GitHub issues (if applicable):

### CODE REVIEW

File: modules/tinymce/src/core/main/ts/api/EditorManager.ts [hunk 1]
```diff
@@ -24,12 +24,15 @@ import URI from './util/URI';
 declare const window: Window & { tinymce: any; tinyMCEPreInit: any };
 
 /**
+ * NOTE: the class tag is commented out for the include in
+ * `modules/tinymce/tools/docs/tinymce.js`
+ *
```
Comment (lnewson) [inline modules/tinymce/src/core/main/ts/api/EditorManager.ts:29]: Can we move this note outside the `EditorManager` class comment block? eg:
```suggestion
// NOTE: the class tag is commented out for the include in `modules/tinymce/tools/docs/tinymce.js`
/**
```

File: modules/tinymce/tools/docs/tinymce.js [hunk 2]
```diff
@@ -14,6 +14,12 @@
  * @property {tinymce.AddOnManager} ThemeManager Global ThemeManager instance. Instance of AddOnManager.
  */
 
+/**
+ * Include EditorManager API docs.
+ *
+ * @include ../../src/core/main/ts/api/EditorManager.ts
```
Comment (lnewson) [inline modules/tinymce/tools/docs/tinymce.js:20]: We should also comment out/remove the `borrows-member` part above as well, as we don't want that to be used if we're including it.

**Review by lnewson — APPROVED** [review]
This turned out better than I expected... I was thinking we'd need a separate file and then include in both locations. However, this keeps the code comments and all lets the API docs work :+1:

File: modules/tinymce/tools/docs/tinymce.js (hunk 2 above)
Comment (tylerkelly13) [reply modules/tinymce/tools/docs/tinymce.js:19]: Deleted in [`b995397` (#6775)](https://github.com/tinymce/tinymce/pull/6775/commits/b99539748bc82c00e104582adc37a9b0db558e01)

File: modules/tinymce/src/core/main/ts/api/EditorManager.ts (hunk 1 above)
Comment (tylerkelly13) [reply modules/tinymce/src/core/main/ts/api/EditorManager.ts]: Fixed in [`b995397` (#6775)](https://github.com/tinymce/tinymce/pull/6775/commits/b99539748bc82c00e104582adc37a9b0db558e01)

**Review by ashfordneil — APPROVED** [review]

**Pull request merged**